frontend/src/components/App/index.js

```diff
@@ -18,7 +18,7 @@ const App = () => {
         <Route exact path={ROUTES.LANDING} component={LandingPage} />
         <Route path={ROUTES.SIGN_IN} component={SignInPage} />
         <Route path={ROUTES.VIEW_TRIPS} component={ViewTripsPage} />
-        <Route path={ROUTES.VIEW_ACTIVITIES + '/:trip-id'} component={ViewActivitiesPage} />
+        <Route path={ROUTES.VIEW_ACTIVITIES + '/:tripId'} component={ViewActivitiesPage} />
```

Comment:
I think you can completely remove the + '/...'  piece because this will be appended inside the ViewTrips component.
```suggestion
        <Route path={ROUTES.VIEW_ACTIVITIES} component={ViewActivitiesPage} />
```

Reply:
No, this is necessary to be able to access the tripID from the URL.